code/modules/antagonists/bloodsucker/datum_bloodsucker.dm

```diff
@@ -199,11 +199,12 @@
 	if(ishuman(owner.current))
 		var/mob/living/carbon/human/H = owner.current
 		var/datum/species/S = H.dna.species
+		var/datum/physiology/physiology
```

Comment:
This is the prooof I never were or will be good enough at explaining how things should be done. 
Remove this line, it will cause null runtimes. :pensive: